wp-content/themes/twentytwenty-child/functions.php

```diff
@@ -17,40 +17,125 @@ function chld_thm_cfg_locale_css( $uri ){
 if ( !function_exists( 'chld_thm_cfg_parent_css' ) ):
     function chld_thm_cfg_parent_css() {
         wp_enqueue_style( 'chld_thm_cfg_parent', trailingslashit( get_template_directory_uri() ) . 'style.css', array(  ) );
+        wp_enqueue_style( 'chld_thm_cfg_child', trailingslashit( get_stylesheet_directory_uri() ) . 'style.css', array(  ) );
```

Comment:
for homework, you need to learn what different arguments in `wp_enqueue_style` are

Reply:
this can be improved, we'll sync